fast_flake_update/__init__.py

```diff
@@ -99,6 +96,20 @@ def main() -> None:
             check=True,
         )
         store_path = res.stdout.strip()
+
+        subprocess.run(
```

Comment:
This should also run in case the previous commands fail.
i.e. one could use contextlib: https://docs.python.org/3/library/contextlib.html
to make sure that even when an exception is thrown the worktree getrs removed.

Reply:
Almost. You also need a finally block so it gets run even when an exception happens.

```
try:
   yield source
finally:
   subprocess.run(...)
```